docker/dockerfiles/zkv-relay.Dockerfile

```diff
@@ -56,8 +56,11 @@ RUN chmod -R a+rx "/usr/local/bin"
 
 RUN apt-get update \
   && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+  aria2 \
```

Comment:
Change line ubuntu version from 22 to 24 
```
FROM ubuntu:24.04 as node
```

Reply:
For the moment we're going to keep 22.04 ; we will bring ubuntu 24.04 into a later dockerfile and entrypoint refactory.